sys-account/server/proto/authn.proto

```diff
@@ -5,19 +5,53 @@ import "google/protobuf/timestamp.proto";
 
 package proto;
 
-option go_package = ".;pb";
+option go_package = ".;rpc";
 
 // Decision decision:
 // success param is when user passes both validation and CSRF token if any
 // ErrorReason is the reason (if any) for failure in processing request
 
 // UserRoles for now
-enum UserRoles {
+enum Roles {
     INVALID = 0;
     GUEST = 1;
     USER = 2;
-    SPECIAL_MEMBER = 3;
-    ADMIN = 4;
+    ADMIN = 3; // can be Org or Project Admin
+    SUPERADMIN = 4; // can do all
+}
+
+// resource: Project
+message Project {
+    string id = 1;
+}
+
+// resource: Org
+message Org {
+    string id = 1;
+}
+
+// UserRole as a message is super simple
```

Comment:
UserRole is defined like this in the protobuf schema, the fields are simple enough in my opinion.

Reply:
@cloudwebrtc  and @WinWisely268 
Don't forget that an Org has many Projects.  
At the moment in the DB we do not model that concept of one Org to many Projects.  
I kept it simple for now you see.  
I am happy to keep it the way it is for now.
But If either of you have thoughts on the good or bad things with this simple approach lets discuss. 


FYI - The biz docs call a Project a "Campaign" but its exactly the same thing, just so you dont get confused when you read their docs.

Reply:
Addressed in #633 and the follow-up comment in the #611